clr-k8s-examples/10-cri-resource-manager/README.md

```diff
@@ -0,0 +1,55 @@
+CRI Resource Manager
+====================
+CRI Resource Manager serves as a relay/proxy between kubelet and the container runtime, relaying requests and responses back and forth between these two, potentially altering requests as they fly by.
+
+This document explains a very simple use case for the `cri-resource-manager`, for more details and tweaks
+on CRI Resource Manager service, you can go to https://github.com/intel/cri-resource-manager.
+
+Install
+-------
+[`install.sh`](install.sh) script will download the binary and install it as an `systemd` service unit. Below you can see the available variables you can use to customize the usage of your CRI Resource Manager service.
+
+| Variable                    | Description                               | Default Value                                    |
+|-----------------------------|-------------------------------------------|--------------------------------------------------|
+| `RUNNER`                    | Default Container Runtime                 | `containerd`                                     |
+| `CRI_RESMGR_POLICY`         | CRI Resource Manager Policy type          | `null`                                           |
+| `CRI_RESMGR_POLICY_OPTIONS` | CRI Resource Manager extra policy options | `-dump='reset,full:.*' -dump-file=/tmp/cri.dump` |
+| `CRI_RESMGR_DEBUG_OPTIONS`  | CRI Resource Manager debugging options    |                                                  |
+
+```
+RUNNER=containerd ./install.sh
+```
+
+- Install verification
+  - Verify that the cri-resource-manager service is actually running.
+```
+systemctl status cri-resource-manager
+```
+  - Verify that the `/var/run/cri-resmgr/cri-resmgr.sock` is created, it will indicate that `cri-resource-manager` is ready to receive requests.
+
+
+Setup as a container runtime in `kubelet`
+----------------------------------------
+The [`setup.sh`](setup.sh) script will configure the `kubelet` service to use the `cri-resource-manager` relay as its remote container runtime.
+```
+./setup.sh
+```
+
+- Setup verification
+  - Kubelet service should be restarted and now using `cri-resource-manager` as its container runtime
```

Comment:
is there any sort of `kubectl` command etc. you can use to verify it is in place? extracting some JSON or something etc.?

Reply:
Tried with the following ones, but they're still showing the botton final container runtime (i.e. containerd, crio or dockershim)

```
kubectl cluster-info dump
kubectl get nodes -o wide
```